spec/porky_lib/unencrypted/file_service_spec.rb

```diff
@@ -145,6 +145,125 @@ def write_test_file(data)
     end
   end
 
+  describe '#write_data' do
```

Comment:
There is a lot of repetition in the tests, maybe we should test write and write_file only for their ability to call write_helper (or write_data if you consider my comment above)

Reply:
Makes sense, it will remove a lot of the repetition of the tests, wasn't sure how to approach it, by combining write_helper into write_data, I will simplify the tests